modules/base/timeframe/timeframeunion.cpp

```diff
@@ -38,30 +38,20 @@ namespace {
         "The time frame is active when any of the contained time frames are, "
         "but not in gaps between contained time frames."
     };
+
+    struct [[codegen::Dictionary(TimeFrameUnion)]] Parameters {
+        // [[codegen::verbatim(TimeFramesInfo.description)]]
+        std::vector<std::monostate> timeFrames [[codegen::reference("core_time_frame")]];
```

Comment:
The `std::monostate` is the only annoying thing left with the codegen ... it's used to generate `ReferenceVerifier` which I was never really happy with.   Whenever a Documentation is tested against and finds a ReferencingVerifier, it looks up the ID in a global database of the DocumentationEngine and finds the documentation for that entry and returns it.   Since the actual type of the thing depends on that Documentation the reference points to, there is no easy type to pick for the variable, so I picked the one std object that can only have a single state;  the monostate.

I'll add something to the documentation, but I'd really like to remove that functionality at some later point instead once we know how to replace it.